cmd/deployment/resource/stop-maintenance.go

```diff
@@ -0,0 +1,74 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmddeploymentresource
+
+import (
+	"fmt"
+
+	"github.com/elastic/cloud-sdk-go/pkg/util/ec"
+	"github.com/spf13/cobra"
+
+	cmdutil "github.com/elastic/ecctl/cmd/util"
+	"github.com/elastic/ecctl/pkg/deployment/depresource"
+	"github.com/elastic/ecctl/pkg/ecctl"
+)
+
+var stopMaintCmd = &cobra.Command{
+	Use:     "stop-maintenance <deployment id> --type <type> [--all|--i <instance-id>,<instance-id>]",
+	Short:   "Stops maintenance mode on a deployment resource",
+	PreRunE: cmdutil.MinimumNArgsAndUUID(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		resType, _ := cmd.Flags().GetString("type")
+		refID, _ := cmd.Flags().GetString("ref-id")
+		instanceID, _ := cmd.Flags().GetStringSlice("instance-id")
+		ignoreMissing, _ := cmd.Flags().GetBool("ignore-missing")
+		all, _ := cmd.Flags().GetBool("all")
```

Comment:
Karen I'm so sorry to post this comment so late! But I think we should by default add a user confirmation prompt when --all is specified warning the user that setting **all** the instances in maintenance mode will cause downtime.

Reply:
nw 😄 I do like the idea of the user confirmation prompt, but it should go in the `start-maintenance` [command](https://github.com/elastic/ecctl/pull/89#discussion_r358003111) and not here. The `stop-maintenance` command "starts" the instances that had been previously put on maintenance mode.

Reply:
d'uh. I missed that